### PR TITLE
add Firefox January email

### PR DESCRIPTION
**Pull request by peiying2**
Per request from bug https://bugzilla.mozilla.org/show_bug.cgi?id=1433498

### CODE REVIEW

File: en-US/emails/2018/jan_mobile_app_updates.lang [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+;So why not get Firefox on your phone or tablet, too? You can sign in to your Firefox Account and your open tabs, bookmarks and passwords will be waiting for you.
+So why not get Firefox on your phone or tablet, too? You can sign in to your Firefox Account and your open tabs, bookmarks and passwords will be waiting for you.
+
+
+;Want more Firefox in more places? You’re in luck:
+Want more Firefox in more places? You’re in luck:
+
+
+;Automatically blocks ads and trackers to keep you safe and browsing fast.
```
Comment (flodolo) [inline en-US/emails/2018/jan_mobile_app_updates.lang]: Explain this is referred to Focus

File: en-US/emails/2018/jan_mobile_app_updates.lang [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+Automatically blocks ads and trackers to keep you safe and browsing fast.
+
+
+# It links users to the app.
+;Get Firefox Focus »
+Get Firefox Focus »
+
+
+;Save articles, videos and more, then read or watch them anywhere without ads, even offline.
```
Comment (flodolo) [inline en-US/emails/2018/jan_mobile_app_updates.lang]: Explain this is referred to Pocket

File: en-US/emails/2018/jan_mobile_app_updates.lang [hunk 3]
```diff
@@ -68,3 +70,8 @@ Have fun and stay safe,
 
 ;The Firefox Team
 The Firefox Team
+
+
+# Link points to https://support.mozilla.org/.
+;P.S. Questions? Problems? Concern? Check out our support site.
```
Comment (flodolo) [inline en-US/emails/2018/jan_mobile_app_updates.lang]: You should indicate which part of the sentence is a link, e.g. 

```
P.S. Questions? Problems? Concern? <a>Check out our support site.</a>
```

File: en-US/emails/2018/jan_mobile_app_updates.lang [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+Have fun and stay safe,
+
+
+;The Firefox Team
+The Firefox Team
+
+
+# The link points to https://support.mozilla.org/.
+;P.S. Questions? Problems? Concern? <a>Check out our support site.</a>
+P.S. Questions? Problems? Concern? Check out our support site.</a>
```
Comment (flodolo) [inline en-US/emails/2018/jan_mobile_app_updates.lang]: Source string and translation are supposed to be identical ;-)

Fixed that, feel free to add other languages.

Comment (peiying2) [reply en-US/emails/2018/jan_mobile_app_updates.lang]: gosh, first i missed the string, then this. :(.

**Pull request merged**